### DISCUSSION TITLE
CosmosDB schema file placement for ACA

### DISCUSSION THREAD

**Discussion by PeerMohamedMydeenA** [Azure/data-api-builder#2137] [Q&A]
I'm trying to deploy DAB on ACA with CosmosDB as Datasource. I placed the schema file (.gql) parallel to the config file in the File Share and I see an error that the schema file is not available. Any recommendations ?

**Comment by seantleonard** [comment]
Thanks for reaching out. Please provide a list of steps/commands that you have completed so far so that we can reproduce in a test environment. 
- Any screenshots and text of error messages will also be helpful.  
- links to specific doc you were following will also be helpful.

**Reply by PeerMohamedMydeenA** [reply]
@seantleonard - Here is the [doc ](https://learn.microsoft.com/en-us/azure/data-api-builder/running-in-azure#use-azure-container-apps) & [sample](https://github.com/Azure/data-api-builder/blob/main/samples/azure/azure-container-apps-deploy.sh) that I was following that worked with MSSQL. This documentation does not have any recommendations/steps specific to schema file placement for CosmosDB.

 DAB-Config
![image](https://github.com/Azure/data-api-builder/assets/134448643/435a8638-bc9e-42c4-8443-c52c3e86737d)

File Share
![image](https://github.com/Azure/data-api-builder/assets/134448643/7ec6278f-33c4-4d2d-ba1c-b24ea1742062)

Error in Logs
![image](https://github.com/Azure/data-api-builder/assets/134448643/7b26ef2b-edcb-4a73-bea2-bc1b0f762765)

Error 
`dime-dab-demo-aca--qpqkp3h update failed. Error : Operation cannot be fulfilled on apps.k8se.microsoft.com "dime-dab-demo-aca--qpqkp3h": StorageError: invalid object, Code: 4, Key: /registry/k8se.microsoft.com/apps/k8se-apps/dime-dab-demo-aca--qpqkp3h, ResourceVersion: 0, AdditionalErrorMsg: Precondition failed: UID in precondition: 69c9d80a-3bd0-44f6-b179-cd4e2dd7439b, UID in object meta:` 

The error message is truncated and could not get the full error message.

**Reply by PeerMohamedMydeenA** [reply]
Looks like I need to update this command with --graphql-schema equivalent ?

/bin/bash, -c, dotnet Azure.DataApiBuilder.Service.dll --ConfigFileName /mnt/dab-config/dime-dab-config.json
